src/components/node/Node.jsx

```diff
@@ -98,9 +100,18 @@ export default class Node extends React.Component {
         let label;
         let node;
 
+        //Determine if we have a rectangle or a square size
+        var sizeWidth = size;
```

Comment:
please use `const` (same on the next line)

Reply:
Updated based on @LonelyPrincess  comments